c-(programming-language)/c-quiz.md

```diff
@@ -1070,3 +1070,28 @@ int main() {
 - [ ] 4
 - [ ] 2
 - [ ] 1
+
+
+ #### Q70. What will be the output of the following code snippet?
+ #include <stdio.h>  
+void change(int,int);  
+int main()  
+{  
+    int a=10,b=20;  
+    change(a,b); //calling a function by passing the values of variables.  
+    printf("Value of a is: %d",a);  
+    printf("\n");  
+    printf("Value of b is: %d",b);  
+    return 0;  
+}  
+void change(int x,int y)  
+{  
+    x=13;  
+    y=17;  
+}  
+
+output --
```

Comment:
```suggestion
 #include <stdio.h>  
void change(int,int);  
int main()  
{  
    int a=10,b=20;  
    change(a,b); //calling a function by passing the values of variables.  
    printf("Value of a is: %d",a);  
    printf("\n");  
    printf("Value of b is: %d",b);  
    return 0;  
}  
void change(int x,int y)  
{  
    x=13;  
    y=17;  
}  

output --
```
you have to put this code inside backtick to make it properly visible in *.md file